CI/travis.linux.after_success.sh

```diff
@@ -32,29 +36,41 @@ if { [ "${DEPLOY}" = "deploy" ]; } ||
     exit
   fi
 
-  # get commit date now before we check out an change into another git repository
+  # get commit date now before we check out and change into another git repository
   COMMIT_DATE=$(git show -s --format="%cs" | tr -d '-')
   YESTERDAY_DATE=$(date -d "yesterday" '+%F' | tr -d '-')
 
-  git clone https://github.com/Mudlet/installers.git "${TRAVIS_BUILD_DIR}/../installers"
+  git clone https://github.com/Mudlet/installers.git "${BUILD_DIR}/../installers"
 
-  cd "${TRAVIS_BUILD_DIR}/../installers/generic-linux"
+  cd "${BUILD_DIR}/../installers/generic-linux"
 
-  ln -s "${TRAVIS_BUILD_DIR}" source
+  ln -s "${BUILD_DIR}" source
 
   # unset LD_LIBRARY_PATH as it upsets linuxdeployqt
   export LD_LIBRARY_PATH=
```

Comment:
I believe to *remove* this from the environment one needs:
```sh
unset LD_LIBRARY_PATH
```
The current script will merely set it to an empty string.